src/routes/demos/activate/.Step5MicrosoftIntegration.svelte

```diff
@@ -0,0 +1,129 @@
+  <div class="column is-full">
+    <div class="level is-mobile">
+      <div class="level-left">
+        <div>
+          <h2 class="subtitle">Step #4 <span class="has-text-info">(optional)</span></h2>
+          <h2 class="title">Microsoft Integration</h2>
+        </div>
+      </div>
+      <div class="level-right">
+        <img class="image is-32x32" src={logo} alt="Microsoft" />
+      </div>
+    </div>
+  </div>
+  <div class="column is-full content mb-0">
+    <p>
+      This is an advanced use case which requires setup of a Microsoft Azure App Registration with Certificates to
+      access the user’s Office 365 calendar. Consult the WXSD team for help/instructions if you want to try this.
+    </p>
+    <p>
+      Once setup, PWA/Kiosk users will be able to join and schedule (create, read, update, delete) personal Microsoft
+      Office 365 events from the device.
+    </p>
+  </div>
+  <div class="column is-full">
+    <label class="label" for="microsoft-tenant-id">
+      Azure Tenant Id
+      <sup class="has-text-danger" title="required" class:is-invisible={!isRequired}>*</sup>
+    </label>
+    <div class="control has-icons-left">
+      <input
+        name="microsoftTenantId"
+        id="microsoft-tenant-id"
+        class="input"
+        type="text"
+        required={isRequired}
+        bind:value={tenantId}
+      />
+      <span class="icon is-left">
+        <i class="mdi mdi-identifier" />
+      </span>
+    </div>
+    <div class="help">
+      <p>This is the id for your Azure Tenant</p>
+    </div>
+  </div>
+  <div class="column is-full">
+    <label class="label" for="microsoft-client-id">
+      Application (client) Id
+      <sup class="has-text-danger" title="required" class:is-invisible={!isRequired}>*</sup>
+    </label>
+    <div class="control has-icons-left">
+      <input
+        name="microsoftClientId"
+        id="microsoft-client-id"
+        class="input"
+        type="text"
+        required={isRequired}
+        bind:value={clientId}
+      />
+      <span class="icon is-left">
+        <i class="mdi mdi-identifier" />
+      </span>
+    </div>
+    <div class="help">
+      <p>This is the client id for your app registration</p>
+    </div>
+  </div>
+  <div class="column is-full">
+    <label class="label" for="microsoft-private-key">
+      Private Key
+      <sup class="has-text-danger" title="required" class:is-invisible={!isRequired}>*</sup>
+    </label>
+    <div class="control has-icons-right">
+      <textarea
+        name="microsoftPrivateKey"
+        id="microsoft-private-key"
+        class="textarea is-family-monospace"
+        rows="7"
+        placeholder={privateKeyPlaceholder}
+        required={isRequired}
+        bind:value={privateKey}
+      />
+      <span class="icon is-right">
+        <i class="mdi mdi-certificate" />
+      </span>
+    </div>
+    <div class="help">
+      <p>This is the private key for your app registration's certificate</p>
```

Comment:
you can you escape out the `'`. it messing up the syntax coloring in github.

Reply:
Thanks. There are other instances of this across the whole code base. Will refactor as part of PR https://github.com/wxsd-sales/roomos-device-widgets/pull/36.